tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -157,4 +167,10 @@ def aggregate(self, tid2tree):
         self.op_list_groupby_name_input = list(agg_result[1].values())
         self.stack_lists_group_by_name = stack_lists_group_by_name
         self.stack_lists_group_by_name_input = stack_lists_group_by_name_input
-        self.tc_eligible_ops = sum([1 if op.tc_eligible else 0 for op in ops])
+        ops_bottom_tc_eligible = []
```

Comment:
better to move all the aggregation for tensorcore into a separated class. For example, TensorCoreAggregator?